package.json

```diff
@@ -35,6 +36,7 @@
   "lint-staged": {
     "*.{ts,json,md}": [
       "prettier --write",
+      "ts-node ./build/stackblitz.ts",
```

Comment:
This will run this command on every changed file, perhaps it would be better to move this to `precommit`?

`"precommit": "lint-staged && npm run build:stackblitz"`

Reply:
👍